user_docs/en/userGuide.t2t

```diff
@@ -2925,6 +2926,30 @@ If you install an add-on with paid components and change your mind about using i
 The Add-on Store is accessed from the Tools submenu of the NVDA menu.
 To access the Add-on Store from anywhere, assign a custom gesture using the [Input Gestures dialog #InputGestures].
 
+++ Note on security and privacy when using Add-ons ++[AddonSecurityAndPrivacy]
+Installing add-ons in NVDA leads to integration of external code into NVDA's functionality in order to enhance NVDA or make new features possible.
+Add-ons can also use external libraries and third party services to serve the purpose and provide the features for which they have been developed. They can be developed by every person or company.
+
+Automated code checks for add-ons happen when they are submitted to NVDA’s Add-on Store. This means e.g.  they will undergo  a basic automated check for vulnerabilities by security analysis tools such as codeQL.
+There isn't currently any formal review process for add-ons performed by a human.
+However, many add-ons have discussions areas where users can exchange feedback. The [community review area #AddonStoreReviews] can be accessed via the actions menu of the add-on.
+
+Installed Add-ons or extensions (not only in NVDA) might in general introduce security and/or privacy vulnerabilities, depending on the permissions they need and actions they perform in order to provide the desired functionality.
+Risks can be e.g.
+- Insecure network connections
+- Files stored with insecure file permissions or in an unprotected location
+- Sensitive information written to an easily available log file
+- Vulnerabilities in third-party libraries
+- Cryptographic vulnerabilities, and more.
+-
+
+Users install NVDA add-ons at their own risk. Therefore, everyone should be aware of following aspects when installing them:
+- Check out the developer’s website to see if it’s a serious source you can trust.
+- Read the description carefully. Does the add-on need questionable permissions? Does it track data? Does it share sensitive data with other sources that you don’t trust?

```

Comment:
What are these permissions you speak of? I'm not asking you to tell me, I'm asking you to explain it to the readers of the user guide who will inevitably be confused by this buzzword.

Additionally, and this is not your fault (it is a common frustration of mine), but many add-on descriptions are absurdly terse to the point of only containing one very brief sentence, which sometimes gives a very inadequate description of what the add-on does (See "ROB enhancements" for an example of this taken to extremes).
Few that I found in just looking at about 100 of them, contain any of the things you say to look for, with the exception of those for ChatGPT, which define what their sharing with by their nature.

Wouldn't it be better to first get @seanbudd to publish some guidelines for what should be in an add-on description, before sticking something in the user guide that says to be suspicious if a description doesn't contain those things?
Or better yet, request to add a field to the add-on metadata which describes what data the add-on accesses, and where it shares it; make it required; and give add-ons time to include that on their next update?

**Any of that should come before this list of concerns are published in the UG.**

```suggestion
- Read the description carefully. Does the add-on need questionable permissions? Does it track data? Does it share sensitive data with other sources that you don’t trust?
```

Reply:
I don't think we should wait for permissions system in NVDA before publishing this. Users who are not sure about these permissions can ask the developer, but there are already a lot of them deductible from add-ons descriptions. e.g. 
* readfeeds add-on uses external links to display feeds and you access those links from NVDA itself when installing the add-on
* The add-on "speak passwords" speaks the characters you press on your keyboard regardless whether they are visible on the screen or not when typed
* NVDA remote add-ons use ports and network connections that in certain cases might be vulnerable
* The link shortener sends your links through is.gd to shorten them, while is.gd has a spam policy for the links, there is still awareness needed that short URLs, when you send them to someone, might imply a process of checking if the destination of the link is the same.
https://www.quora.com/How-can-you-tell-if-a-shortened-URL-is-legitimate
Ofcourse not every add-on author has some indications in their descriptions, but in many add-on descriptions such things are deductible by reading carefully and having a general knowledge about permissions.
This term is something that is very common, same like e.g. "component".  We don't have to explain it in the user guide.